modules/apps/portal-search/portal-search-admin-web/src/main/resources/META-INF/resources/index_actions.jsp

```diff
@@ -329,7 +344,14 @@ page import="java.util.Map" %>
 										</div>
 
 										<div class="autofit-col index-action-wrapper" data-type="<%= indexReindexerClassName %>">
-											<aui:button cssClass="save-server-button" data-classname="<%= indexReindexerClassName %>" data-cmd="reindexIndexReindexer" data-concurrent-disabled="" value="execute" />
+											<c:choose>
```

Comment:
there is probably(?) a way to extract this since its basically duplicating the code above, @oliv-yu might have some input, also since there are probably going to be conflicts with the code that was just merged yesterday https://github.com/brianchandotcom/liferay-portal/pull/139772

Reply:
Oh thank you for working on this, I was curious about the background tasks not being available for `reindexIndexReindexer`. All the changes for the new reindex page design + sync + concurrent are behind `LPS-183661`, but there is still the conditional for the original, so it looks like it won't cause merge conflicts.

Before you send, would you mind rebasing off master (the new layout should be on there) and cherry-picking my last commit to your branch? It's just to apply your background tasks to the new layout. https://github.com/oliv-yu/liferay-portal/commits/LPS-194426

Thanks!